.azuredevops/azure-pipelines.yml

```diff
@@ -58,28 +57,30 @@ steps:
   - script: flutter analyze
     displayName: Analyze code
 
-  - script: flutter build
```

Comment:
Don't we need this build to validate that the code compiles?

Reply:
updated